etc/encoding/adaptive-streaming-movies.properties

```
@@ -40,7 +40,7 @@ profile.adaptive-1080p.http.suffix = -preview.mp4
 profile.adaptive-1080p.http.ffmpeg.command = -i #{in.video.path} \
   -c:v libx264 -crf 23 -maxrate 4000k -bufsize 8000k -profile:v high -level 4.0 -pix_fmt yuv420p \
   -x264opts keyint=25:min-keyint=25:no-scenecut -movflags +faststart \
-  -filter:v scale='-2:min(ih\\,1080)',fps=fps=25 \
+  -filter:v crop='floor(iw/2)*2:floor(ih/2)*2',scale='-2:min(ih\\,1080)',fps=fps=25 \
```

Wouldn't `force_original_aspect_ratio=decrease` and `force_divisible_by=2` make way more sense than chaining two scale filters? I know that the latter option is somewhat new but the release of 9.x is also quite some time off.

Essentially:
```suggestion
  -filter:v crop=1920:1080:force_original_aspect_ratio=decrease:force_divisible_by=2,fps=25 \
```

---

Technically this does a crop and a scale, not two scales. I compared execution times with vs without the crop.

Without crop:
```
real    2m3.469s
user    3m38.199s
sys     0m1.443s
```

With crop:
```
real    1m56.107s
user    3m34.956s
sys     0m1.481s
```

So the crop operation doesn't seem to make a difference performance-wise. Probably because nothing is cropped when using a video with an even width and height.

(Your edit should probably say `scale=`, not `crop=`.)

I tried to use `force_divisible_by` but on my CentOS 7 machine neither the package version of ffmpeg nor the statically linked one from https://johnvansickle.com/ffmpeg/ support it. So I couldn't test that yet.

---

Just wondering: Do you want to get this into 8.x (maybe even 8.3)? If yes, then rebase this against `r/8.x` and we can leave it like this. The feature I mentioned is too new for that and this could be counted as a bug fix. Note that 8.3 should be released this morning (CET).

---

If this is targeted at `develop`:

It's not in the stable but it is in the master build of johnvansickle.com/ffmpeg and it's also in the release which is in Opencast's package repo.

With the target release for this probably ending up to be in December, that's more than enough time for that feature to settle and I just think that that filter chain makes more sense ;-)

---

Ok. I rebased this PR against `r/8.x` and made a new one with your suggested change for `develop`: #1490